pkgs/top-level/all-packages.nix

```diff
@@ -18667,7 +18667,8 @@ with pkgs;
   texinfo6_5 = callPackage ../development/tools/misc/texinfo/6.5.nix { }; # needed for allegro
   texinfo6_7 = callPackage ../development/tools/misc/texinfo/6.7.nix { }; # needed for gpm, iksemel and fwknop
   texinfo6 = callPackage ../development/tools/misc/texinfo/6.8.nix { };
-  texinfo = texinfo6;
+  texinfo7 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
+  texinfo = texinfo7;
```

Comment:
I suggest to put this change of default texinfo into a separate commit. This way we can guarrantee each commit is buildable.

Reply:
I have corrected the typo in the commit message—thanks for noticing and pointing it out—and divided the addition of texinfo 7.0.2 and making that the default version into separate commits.

What are your thoughts on [@trofi's comment](https://github.com/NixOS/nixpkgs/pull/215699#pullrequestreview-1293873954), @AndersonTorres?

Reply:
> This way we can guarrantee each commit is buildable.

Then ``texinfo: Add version 7.0.2`` and ``texinfo: Add postPatch with patchShebangs to common.nix`` would need to be combined, right?

Reply:
✅ I was uncertain about it, but it does make sense.